spec.html

```diff
@@ -16371,7 +16371,7 @@ <h1>White Space</h1>
       <p>U+0020 (SPACE) and U+00A0 (NO-BREAK SPACE) code points are part of &lt;USP>.</p>
     </emu-note>
     <emu-note>
-      <p>Other than for the code points listed in <emu-xref href="#table-white-space-code-points"></emu-xref>, ECMAScript |WhiteSpace| intentionally excludes all code points that have the Unicode “White_Space” property but which are not classified in general category “Space_Separator” (“Zs”).</p>
+      <p>Other than the code points listed in <emu-xref href="#table-white-space-code-points"></emu-xref>, ECMAScript |WhiteSpace| intentionally excludes all code points that have the Unicode “White_Space” property but which are not classified in general category “Space_Separator” (“Zs”). As of Unicode 15.1.0, the only such code point that is not a <emu-xref href="#sec-line-terminators">line terminator</emu-xref> is U+0085 (NEXT LINE).</p>
```

Comment:
The editor group wasn't happy with this change as-is. If we really wanted to add something for the readers who might be already familiar with Unicode's `White_Space` property who may be confused, we can say something like

```suggestion
      <p>Other than the code points listed in <emu-xref href="#table-white-space-code-points"></emu-xref>, ECMAScript |WhiteSpace| intentionally excludes all code points that have the Unicode “White_Space” property but which are not classified in general category “Space_Separator” (“Zs”), such as all <emu-xref href="#table-line-terminator-code-points">line terminators</emu-xref>, U+0085 (NEXT LINE), and possibly others.</p>
```

But that's also probably not a significant population of readers, so it's also okay to not add anything here. Closing for now.